crates/consensus/src/transaction/mod.rs

```diff
@@ -100,18 +99,6 @@ pub trait SignableTransaction<Signature>: Transaction {
     fn into_signed(self, signature: Signature) -> Signed<Self, Signature>
     where
         Self: Sized;
-
-    /// Encode with a signature. This encoding is usually RLP, but may be
-    /// different for future EIP-2718 transaction types.
-    fn encode_signed(&self, signature: &Signature, out: &mut dyn BufMut);
```

Comment:
do we not need this?

Reply:
`encode_signed` and `decode_signed` are not specific enough for eth transactions, it's ambiguous whether or not you want the network or RPC RLP format for transactions. Encodable2718 provides this distinction

Reply:
ok, makes sense to me. need to document this, but can be a follow up, would create an issue